src/nvim/api/ui_events.in.h

```diff
@@ -175,6 +175,9 @@ void msg_ruler(Array content)
 void msg_history_show(Array entries, Boolean prev_cmd)
   FUNC_API_SINCE(6) FUNC_API_REMOTE_ONLY;
 
+void tty_write(String content)
```

Comment:
nit: the events in this file are somewhat "grouped"; the global events are near the top.